### DISCUSSION TITLE
[Feature] "Go to date" from side info panel

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#10460] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Currently, clicking on the date only triggers editing, on the right-side info panel of a given picture.
Instead, it would be nice to **also** be able to jump to a filtered search for that very specific date, just to place the picture back in context in case it was not accessed through the chronological list.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#4115
